v2/diff_write.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"fmt"
 
+	lcs "github.com/yudai/golcs"
```

Comment:
https://github.com/yudai/golcs seems deprecated, or at least unmaintained

Did you consider other libraries such as these ones?

https://github.com/vmarkovtsev/go-lcss
https://github.com/peter-evans/patience

These are examples of library I found by taking a look at available libraries

https://github.com/search?q=language%3Ago+lcs&type=repositories&s=updated&o=desc

Reply:
github.com/yudai/golcs is the lcs library used in other places in the codebase of `jd` (see `v2/list.go`)

Reply:
Indeed, otherwise the go.mod would have been updated 😅

So, it's of course normal to use it.